service/controller/aws/v11/version_bundle.go

```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "cluster-operator",
+				Description: "Reduce installation time of managed apps in new tenant clusters.",
```

Comment:
@teemow @puja108 I went for this as the changelog. Does it look OK?

Reply:
:+1: